src/core/dbus-cgroup.c

```diff
@@ -714,6 +717,9 @@ int bus_cgroup_set_property(
 
                 while ((r = sd_bus_message_read(message, "(st)", &path, &weight)) > 0) {
 
+                        if (!path_startswith(path, "/dev"))
```

Comment:
hmm, we should allow path_startswith(path, "/run/systemd/inaccessible/") here too I figure, as we create 0:0 device nodes there and those should be includable in a device policy